src/slurm_plugin/resume.py

```diff
@@ -161,8 +161,12 @@ def _resume(arg_nodes, resume_config):
         _handle_failed_nodes(arg_nodes)
         return
     log.info("Launching EC2 instances for the following Slurm nodes: %s", arg_nodes)
-    node_list = [node.name for node in get_nodes_info(arg_nodes)]
-    log.debug("Retrieved nodelist: %s", node_list)
+    node_list = []
+    node_list_with_status = []
```

Comment:
have you evaluated the pros/cons to have just one list?

Reply:
The `node_list` is used for `add_instances_for_nodes` by both `clustermgtd.py` and `resume.py`.

In the `clustermgtd.py` the `node_list` is produced in a different way. I prefer to not touch the logic of the `add_instances_for_nodes` just to improve the logging for now. This change can be evaluated separately.